batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/controller/IndexController.java

```diff
@@ -16,5 +16,5 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         Integer i = TypeCaster.toInt("123");
         System.out.println("value of i: " + i);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)